app/connectors/SchemeDetailsConnector.scala

```diff
@@ -70,7 +70,7 @@ class SchemeDetailsConnectorImpl @Inject() (appConfig: FrontendAppConfig, http:
     val headers = List("pspId" -> pspId.value, "srn" -> schemeId.value)
 
     http
-      .get(url("/pensions-scheme/psp-scheme"))
+      .get(url(s"/pensions-scheme/psp-scheme/${schemeId.value}"))
       .setHeader(headers*)
       .execute[Option[SchemeDetails]]
       .tapError { t =>
```

Comment:
Could you please check what happens if the user is not associated with the scheme?
Expected: show some kind of unauthorised page.
I think this will just throw a generic exception and result in a generic "Problem with service" page.

Reply:
yes, but what will PSR SIPP FE display?

Reply:
Great, thanks!